product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/monitoring_from_azure/index.mdx

```diff
@@ -53,6 +53,49 @@ PostgresAuditLogs_CL
 | project record_log_time_s, record_error_severity_s, record_message_s
 | sort by record_log_time_s desc
 ```
+ 
+### Using BigAnimal's cloud account
+
+To access your Postgres cluster logs, when using BigAnimal's cloud account, generate a SAS token from BigAnimal and use it to download the logs. 
+
+1. In the BigAnimal portal, select **Clusters**, select your cluster, and select the **Monitoring & Logging** tab. 
+
+1. Select **Generate Token** and copy the SAS token. The SAS token is a sensitive value and shouldn't be made publicly available. The following is a sample SAS token: 
+
+   ```
+   https://blobsamples.blob.core.windows.net/?sv=2022-11-02&ss=b&srt=sco&sp=rwlc&se=2023-05-24T09:51:36Z&st=2023-05-24T01:51:36Z&spr=https&sig=<signature> 
+   ```
+    
+1. Enter the `azcopy` command to download the Postgres logs from BigAnimal. For example:
+
+   ```
+   azcopy copy '$TOKEN' . --recursive
+   INFO: Scanning...
+   INFO: Any empty folders will not be processed, because source and/or destination doesn't have full folder support
+   
+   Job aa4b74a0-bc92-be4e-551c-47aec1c1cfc3 has started
+   Log file is located at: /Users/valeriodelsarto/.azcopy/aa4b74a0-bc92-be4e-551c-47aec1c1cfc3.log
+   
+   100.0 %, 5 Done, 0 Failed, 0 Pending, 0 Skipped, 5 Total, 2-sec Throughput (Mb/s): 0.5375
+   
+   
+   Job aa4b74a0-bc92-be4e-551c-47aec1c1cfc3 summary
+   Elapsed Time (Minutes): 0.0333
+   Number of File Transfers: 5
+   Number of Folder Property Transfers: 0
+   Number of Symlink Transfers: 0
+   Total Number of Transfers: 5
+   Number of File Transfers Completed: 5
+   Number of Folder Transfers Completed: 0
+   Number of File Transfers Failed: 0
+   Number of Folder Transfers Failed: 0
+   Number of File Transfers Skipped: 0
+   Number of Folder Transfers Skipped: 0
+   TotalBytesTransferred: 134416
+   Final Job Status: Completed
+   $ tail p-456g715b75/kubernetes-logs/p-456g715b75/2023/09/26/13/19/azure_customer_postgresql_cluster.var.log.containers.p-456g715b75-1_p-456g715b75_postgres-c798aa19ea0481c8d9575f025405b3ad9212816ca7e928f997473055499a692c.log
```

Comment:
should we use a clearly fake PG cluster ID instead of this one that was really existing? Something like `p-a1b2c3d4d5` or it's fine like this?

Reply:
Changed it to what you suggested.